### PR TITLE
Warn validators with slow hardware

### PR DESCRIPTION
**Pull request by Szegoo**
Companion for: https://github.com/paritytech/substrate/pull/12620

polkadot companion: https://github.com/paritytech/polkadot/pull/6269

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2289467

**Comment by ggwpez** [comment]
bot rebase

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2292332

**Review by ggwpez — APPROVED** [review]

**Review by sam0x17 — APPROVED** [review]

File: polkadot-parachain/src/service.rs [hunk 1]
```diff
@@ -1580,3 +1589,15 @@ pub async fn start_contracts_rococo_node(
 	)
 	.await
 }
+
+/// Checks that the hardware meets the requirements and print a warning otherwise.
+fn warn_if_slow_hardware(hwbench: &sc_sysinfo::HwBench) {
+	// Polkadot para-chains should generally use these requirements to ensure that the relay-chain
+	// will not take longer than expected to import the its blocks.
```
Comment (ggwpez) [inline polkadot-parachain/src/service.rs:1596]:
```suggestion
	// will not take longer than expected to import its blocks.
```

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2295501

**Review by bkchr — APPROVED** [review]

**Comment by muharem** [comment]
bot rebase

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2297035

**Pull request merged**